package.json

```diff
@@ -1,123 +1,123 @@
-        "husky": "^8.0.0",
-        "identity-obj-proxy": "^3.0.0",
-        "jest": "^28.1.3",
-        "jest-cli": "^28.1.3",
-        "jest-environment-jsdom": "^28.1.3",
-        "lerna": "^5.6.1",
-        "openmrs": "next",
-        "plotly.js": "^2.24.3",
-        "prettier": "^2.8.8",
-        "pretty-quick": "^3.1.3",
-        "raw-loader": "^4.0.2",
-        "react": "^18.2.0",
-        "react-csv": "^2.2.2",
-        "react-dom": "^18.2.0",
-        "react-i18next": "^11.18.6",
-        "react-pivottable": "^0.11.0",
-        "react-plotly.js": "^2.0.0",
-        "react-router-dom": "^6.11.2",
-        "react-table": "^7.8.0",
-        "rxjs": "^6.6.7",
-        "swc-loader": "^0.2.3",
-        "turbo": "^1.10.12",
-        "typescript": "^4.9.5",
-        "webpack": "^5.88.1",
-        "webpack-cli": "^5.1.3"
-    },
-    "resolutions": {
-        "@carbon/react": "1.14.0"
-    },
-    "packageManager": "yarn@3.6.3"
+  "name": "@openmrs/esm-laboratory-app",
+  "version": "1.0.0",
+  "license": "MPL-2.0",
+  "description": "Laboratory microfrontend for OpenMRS 3.x",
+  "browser": "dist/openmrs-esm-laboratory-app.js",
+  "main": "src/index.ts",
+  "source": true,
+  "scripts": {
+    "start": "openmrs develop",
+    "serve": "webpack serve --mode=development",
+    "build": "webpack --mode production",
+    "analyze": "webpack --mode=production --env analyze=true",
+    "lint": "TIMING=1 eslint src --ext js,jsx,ts,tsx",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx}\"",
+    "typescript": "tsc",
+    "test": "jest --config jest.config.js --passWithNoTests",
+    "verify": "turbo lint typescript test",
+    "extract-translations": "i18next 'src/**/*.component.tsx' --config ./i18next-parser.config.js",
+    "coverage": "yarn test -- --coverage",
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && yarn verify"
+    }
+  },
+  "browserslist": [
+    "extends browserslist-config-openmrs"
+  ],
+  "keywords": [
+    "openmrs",
+    "microfrontends",
+    "laboratory"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openmrs/openmrs-esm-laboratory.git"
+  },
+  "homepage": "https://github.com/openmrs/openmrs-esm-laboratory#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/openmrs/openmrs-esm-laboratory/issues"
+  },
+  "dependencies": {
+    "@carbon/react": "^1.14.0",
+    "@hookform/resolvers": "^3.3.4",
+    "lodash-es": "^4.17.21",
+    "react-hook-form": "^7.49.3",
+    "react-to-print": "^2.14.15",
+    "zod": "^3.22.4"
+  },
+  "peerDependencies": {
+    "@openmrs/esm-framework": "*",
+    "dayjs": "1.x",
+    "react": "18.x",
+    "react-i18next": "11.x",
+    "react-router-dom": "6.x",
+    "rxjs": "6.x"
+  },
+  "devDependencies": {
+    "@ohri/openmrs-esm-ohri-commons-lib": "next",
+    "@openmrs/esm-extensions": "next",
+    "@openmrs/esm-framework": "^5.3.2",
```

Comment:
Was this intentional? or did yarn do this? typically we use `next` tag so even after the `yarn up` it's better to leave it as `next`

Reply:
Yarn did this. Let me get it back